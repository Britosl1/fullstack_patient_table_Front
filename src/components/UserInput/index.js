import React from 'react';
import { FormControl } from 'react-bootstrap'

function UserInput({ searchValue, onChange }) {
  return (
    <FormControl
      value={searchValue}
      onChange={onChange}
      type="text" 
      placeholder="Search for user name or nationality..."
      style={{ marginBottom: '20px'}} />
  );
}

export default UserInput;