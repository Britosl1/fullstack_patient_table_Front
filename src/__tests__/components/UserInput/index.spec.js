import React from 'react';
import renderer from 'react-test-renderer';
import UserInput from '../../../components/UserInput/index';

describe('Snapshot test', () => {
  it('renders correctly', () => {
    const nav = renderer
      .create(<UserInput searchValue="Teste" onChange={jest.fn()}/>)
      .toJSON();
    expect(nav).toMatchSnapshot();
  });

})