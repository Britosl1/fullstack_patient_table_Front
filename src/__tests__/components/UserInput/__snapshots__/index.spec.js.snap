// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test renders correctly 1`] = `
<input
  className="form-control"
  onChange={[MockFunction]}
  placeholder="Search for user name or nationality..."
  style={
    Object {
      "marginBottom": "20px",
    }
  }
  type="text"
  value="Teste"
/>
`;
